README.md

```diff
@@ -14,6 +14,7 @@ GEI-CLI is continuing to expand what it can support. However, it supports the fo
 
 * Azure DevOps -> GitHub Enterprise Cloud migrations
 * GitHub Enterprise Cloud -> GitHub Enterprise Cloud migrations
+* GitHub Enterprise Server (> version 3.4) -> GitHub Enterprise Cloud migrations
```

Comment:
```suggestion
* GitHub Enterprise Server (version 3.4 or newer) -> GitHub Enterprise Cloud migrations
```

Reply:
I'd make this more clear as some customers get confused with `>` and `<` in the past experience.